src/nimlsppkg/suggestlib.nim

```diff
@@ -5,8 +5,10 @@ import compiler / ast
 export Suggest
 export IdeCmd
 export NimSuggest
+import os
```

Comment:
New import without any new functionality?

Reply:
this file is irrelevant,I tried to rewrite initNimsuggest for making it accept project root argument,then I checked the nimsuggest code , not necessary,